Packs/ZeroFox/ReleaseNotes/1_0_6.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### ZeroFox
+- Updated the Docker image to: *demisto/python3:3.10.7.33922*.
+- Fixed an issue where only updated alerts were fetched with *fetch_incidents*.
```

Comment:
Only updated alerts were fetched? or do both new and updated alerts created new incidents?

Reply:
You're right. I'll rephrase it better.